webapp/src/components/PublishParcelPage/PublishParcelPage.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import { Link } from 'react-router-dom'
-import { Loader, Container, Message } from 'semantic-ui-react'
+import { Loader, Container, Message } from 'semantic-ui-react/dist/commonjs'
```

Comment:
Careful here! the old version is the correct import

Reply:
F#ck this is related to what I told you when I started using vs code, imports gets crazy